### DISCUSSION TITLE
Ì cannot get IntuneCD-startupdate  to update the configuration in intune/Entra

### DISCUSSION THREAD

**Discussion by Kumara-Upadhayayula** [almenscorner/IntuneCD#150] [Q&A] [answered]
``I have created an Graph application and followed all the wiki, I can create the documentation and make an backup of the currents environment but i am unable to perform an restore to the backup of the environment

I am using the Auth.json to store all the values required for authentication in the following format
{
    "params":{
        "TENANT_NAME": "",
        "CLIENT_ID": "",
        "CLIENT_SECRET": ""
    }
}
![Screenshot 2023-11-12 141203](https://github.com/almenscorner/IntuneCD/assets/95960631/91ea2454-431f-4b28-8e42-df4a3c02e756)

`IntuneCD-startupdate -p 'C:\Users\c150831_a\Desktop\test\' -m 1 -a 'C:\Users\c150831_a\Desktop\Powershell_intro\Graph_Dev_Tennant\04RestoreConfig\auth.json' -r`

The backup is performed as expected but it will not do anything when I run the command , when i run in report only mode it will say no changes need to be uploaded into intune

**Comment by almenscorner** [comment]
Hi, I'm not sure I'm following what the issue is here. Do you mean that you have the backup, changes are present in the backup but when you run the update these changes are not pushed to Intune?

Are you using multiple tenants?
Have you verified there actually is a change in the backup?

**Reply by Kumara-Upadhayayula** [reply]
Hi there I have made backup changes in the UI by deleting some conditional access policies as well as device configuration policies and tried restoring the policies from intuneCD using the patch to the backups, the backups are from an single prod tenant and I'm trying to restore them on the same Tenant
I am also using windows for testing the deployment

**Reply by almenscorner** [reply]
What is the exact command you run when trying to restore?

**Reply by Kumara-Upadhayayula** [reply]
IntuneCD-startupdate.exe -p C:\Users\c150831_a\Desktop\test -m 1 -a C:\Users\c150831_a\Desktop\Powershell_intro\Graph_Dev_Tennant\04RestoreConfig\auth.json

**Reply by almenscorner** [reply]
So is this the flow you are trying to make work?

- Run a backup
- Remove configurations in Intune
- Run IntuneCD update to restore them

**Reply by Kumara-Upadhayayula** [reply]
Yup that's correct

**Reply by almenscorner** [reply]
What is the output from IntuneCD when you run the update?

**Reply by Kumara-Upadhayayula** [reply]
It doesn't have any output just returns nothing 

![Screenshot 2023-11-13 203736](https://github.com/almenscorner/IntuneCD/assets/95960631/ed1a6647-17dd-434c-9923-e45b14c03223)

**Reply by almenscorner** [reply, marked as answer]
This was definitely a bug, I have pushed a bug fix. Update to 2.0.7 in a couple of minutes and try the update again

**Reply by Kumara-Upadhayayula** [reply]
Thank you for that :)

**Reply by Kumara-Upadhayayula** [reply]
Thank you , I can now restore from the backup now
